src/test/java/org/eclipse/tracecompass/trace_event_logger/res/badlogging.properties

```diff
@@ -0,0 +1,8 @@
+handlers= org.eclipse.tracecompass.trace_event_logger.SnapshotHandler, org.eclipse.tracecompass.trace_event_logger.AsyncFileHandler
```

Comment:
is this file referenced anywhere? Not sure why it's there.

Reply:
Explained in code, but it's used for unit tests